core/src/main/java/com/epam/esm/dao/entity/Tag.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.esm.dao.entity;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/** Entity class for creating {@code Tag} objects. */
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "tag")
+public class Tag implements Serializable {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@EqualsAndHashCode.Exclude
+	private long id;
+
+	@Column(name = "name")
+	private String name;
+
+	public Tag(String name) {
+		this.name = name;
+	}
+
+	public Tag(long id, String name) {
+		this.id = id;
+		this.name = name;
+	}
+
+	@ManyToMany(mappedBy = "tags")
+	private Set<Certificate> certificates = new HashSet<>();
+
+	@Override
+	public boolean equals(Object o) {
+		if (this == o)
```

Comment:
please add brackets of classic code conventions
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
page 17

Reply:
Done